provisioner/template.go

```diff
@@ -512,9 +512,12 @@ func amiID(adapter *awsAdapter, imageName, imageOwner string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("failed to describe image with name %s and owner %s: %v", imageName, imageOwner, err)
 	}
-	if len(output.Images) != 1 {
+	if len(output.Images) > 1 {
 		return "", fmt.Errorf("more than one image found with name: %s and owner: %s", imageName, imageOwner)
 	}
+	if len(output.Images) < 1 {
```

Comment:
We could check for `0` here since it cannot be negative. It might be easier to understand this way.